ping.go

```diff
@@ -179,7 +182,7 @@ type Packet struct {
 	Seq int
 
 	// TTL is the Time To Live on the packet.
-	Ttl int
+	TTL int
```

Comment:
I know it's only recently been merged, but this is now a breaking change and I'm not sure it's worth it just for the sake of linting.

Reply:
Not saying it's correct, but as Ttl it's also consistent with Rtt.